### ISSUE TITLE
mower gets invisible at some orientations

### ISSUE THREAD

**Issue by disaster123** [EinEinfach/CaSSAndRA#69]
regulary the mower get's invisible / has no icon at some orientation.

**Comment by ShadedSelf** [comment]
Currently there is one image per robot rotation, I think the image should be able to be rotated instead with time image.rotate() function.
I might look into this at some point, unless @EinEinfach or someone else wants to do it first.

**Comment by EinEinfach** [comment]
> Currently there is one image per robot rotation, I imagine there is one missing.
> I think we can have one image and just rotate it with image.rotate() function instead .
> I might look into this at some point, unless @EinEinfach or someone else wants to do it first.

Good idea!

**Comment by EinEinfach** [comment]
I commited some changes related to this issue. Pls test.

**Comment by rothnic** [comment]
@disaster123 are you still seeing this issue? I think I was seeing that at times, but lately haven't noticed it.

**Comment by disaster123** [comment]
I had too bad wheather the last days. Will need to check this the next days. Basically i can tell you that I'm still seeing the robot sometimes rotated by 180 degrees. So in some orientation it drives suddenly backwards in cassandra while driving forward in real.

**Comment by rothnic** [comment]
Yeah, I think I have also seen it facing the wrong way when docked, in particular. I just started a mow task from the taskplanner page a moment ago, and when I went to the main page, I noticed I didn't see the mower at all. I could see the next point it was going to, but the mower wasn't shown. I tried refreshing the whole page, and it still isn't shown.

<img width="1428" alt="Screen Shot 2023-09-24 at 12 46 48 PM" src="https://github.com/EinEinfach/CaSSAndRA/assets/452052/86241ea2-3576-4e33-a503-f168c1e02738">

**Comment by disaster123** [comment]
+1 same still here

**Comment by rothnic** [comment]
Just noticed there has been another commit about the rover images from yesterday, and I haven't tried that out yet. Also wanted to note that mine did eventually show on this mow task, but it isn't clear what caused it to show again.

**Comment by EinEinfach** [comment]
Are you use last version of Cassandra?

**Comment by disaster123** [comment]
This is with latest master:
![image](https://github.com/EinEinfach/CaSSAndRA/assets/861558/032d1e22-4349-4b5c-8d2b-10e2e102e894)

mower is currently docked and invisible....

**Comment by EinEinfach** [comment]
Try to change mower image in settings section. Is the mower visible after that? If yes change it again to wanted image, maybe is something gone wrong at start up

**Comment by disaster123** [comment]
no does not help. I switched the image multiple times - mower stays invisible until it moves / rotate again

**Comment by EinEinfach** [comment]
Can you increase PIL logger level,  maybe then I can see more

**Comment by EinEinfach** [comment]
Is the mower also invisible on mapping page?
